packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCommunitySelector/CWCommunitySelector.tsx

```diff
@@ -5,20 +5,33 @@ import { ComponentType } from '../../types';
 import { CWText } from 'views/components/component_kit/cw_text';
 import { CWTag } from 'views/components/component_kit/new_designs/CWTag';
 
+import { ChainBase } from 'common-common/src/types';
 import './CWCommunitySelector.scss';
 
-type CommunityType = 'ethereum' | 'cosmos' | 'polygon' | 'solana';
+export enum CommunityType {
+  Ethereum = 'ethereum',
+  Cosmos = 'cosmos',
+  Polygon = 'polygon',
+  Solana = 'solana',
+}
+
+export type SelectedCommunity = {
+  type: CommunityType;
+  chainBase: ChainBase;
+};
 
 interface CWCommunitySelectorProps {
   type: CommunityType;
+  chainBase: ChainBase;
   title: string;
   isRecommended?: boolean;
   description: string;
-  onClick: (type: CommunityType) => void;
+  onClick: ({ type, chainBase }: SelectedCommunity) => void;
```

Comment:
I guess same issue with community type step. The function arguments don't change in the child component, we should probably curry them in the parent function and remove the unused props.

Reply:
yup, good catch. Changed